src/test/resources/org/rust/lang/core/parser/fixtures/complete/expr.rs

```diff
@@ -80,6 +80,7 @@ fn test() -> u32 {
     let _ = 1f34;
     let _ = 1.0i98;
     let _ = 0.0.0;
+    let _ = 0x1234567890abcdef;
```

Comment:
Maybe it makes sense to extract literal parsing into separate tests?
Also, it would be great to add binary and octal integer literals as well

Reply:
Done